build/git-history.js

```diff
@@ -35,17 +36,20 @@ function getFromGit(contentRoot = CONTENT_ROOT) {
   );
 
   const map = new Map();
-  let date = null;
+  let date = null,
+    hash = null;
```

Comment:
Nit: I don't like this notation. Now the two variables get entangled. A change to one might cause the git diff to touch both lines. I'm surprised `eslint` didn't complain. 
Most of the time, in this code base, you'll find notation like:
```javascript
let date = null;
let hash = null;
```